spec/reek/report/location_formatter_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../../spec_helper'
+require_relative '../../../lib/reek/report/formatter'
+require_relative '../../../lib/reek/smells/duplicate_method_call'
+require_relative '../../../lib/reek/smells/smell_warning'
```

Comment:
Not something specific to this pull request but I believe the "../../../" madness in our specs highly distracting.
How about setting up a spec wide visible constant that points to $ROOT/lib/reek?
I believe this would clean up things quite a bit...